inductiva/fluids/post_processing/openfoam.py

```diff
@@ -93,6 +93,7 @@ def get_streamlines(self,
                         n_points: int = 100,
                         initial_step_length: float = 1,
                         source_radius: float = 0.7,
+                        source_center: tuple = (0, 0, 0),
```

Comment:
does this allow only 3d tuples? if so, you can type annotate tuple[float, float, float], this way you're telling that the tuple needs 3 elements